.github/pull_request_template.md

```diff
@@ -7,5 +7,4 @@
 ### Checklist:
 - [ ] Tests added
 - [ ] CHANGELOG entry added 
-  > HashiCorp engineers only, community PRs should not add a changelog entry.
   > Entries should use imperative present tense (e.g. Add support for...)
```

Comment:
Can we add a note here about the changelog location and naming convention? It isn't clear to me outside of reading the Actions workflow what I'm supposed to do as a PR author.